packages/types/src/package.ts

```diff
@@ -171,6 +171,13 @@ export default Package;
 /**
  * An individual Frontity action.
  */
+// NOTE: This interface looks like was intended to be used in the `Actions`
+// interface defined below, but it creates a circular reference and breaks it,
+// so it's currently unused.
+//
+// TODO: either fix these types or remove this `Action` definition.
+//
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do you have more context for that comment? Maybe we should delete that interface?

Reply:
I think we should remove it (it's not being used because it doesn't work) and also fix the TSDocs in `action.ts`.

Reply:
Ok, go ahead then 🙂